plugins/chat/app/jobs/regular/send_message_notifications.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+  class SendMessageNotifications < ::Jobs::Base
+    def execute(args)
+      reason = args[:reason]
```

Comment:
Shouldnt we return if we have no reason?

Reply:
There's no `else` in the `if` so nothing will happen if reason is not "new" or "edit"

Reply:
Yes but still we don't want to do any work